flink-connectors/flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/JdbcConnectionOptions.java

```diff
@@ -92,8 +99,13 @@ public JdbcConnectionOptionsBuilder withPassword(String password) {
 			return this;
 		}
 
+		public JdbcConnectionOptionsBuilder withConnectionCheckTimeoutSeconds(int connectionCheckTimeoutSeconds) {
```

Comment:
Could you add a javadoc on this method? Because this is a public API.

Reply:
I don’t know a standard statement about this API

Reply:
similar to the description in the doc.

Reply:
ok, I tried to add some